plugins/application/loki/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/infrawatch/apputils/connector"
+	"github.com/infrawatch/apputils/logging"
+	"github.com/infrawatch/sg-core/pkg/application"
+	"github.com/infrawatch/sg-core/pkg/config"
+	"github.com/infrawatch/sg-core/pkg/data"
+	"github.com/pkg/errors"
+
+	"github.com/infrawatch/sg-core/plugins/application/loki/pkg/lib"
+)
+
+type LokiConfig struct {
+	Connection  string `validate:"required"`
+	BatchSize   int64
+	MaxWaitTime time.Duration
+}
+
+//Loki plugin for forwarding logs to loki
+type Loki struct {
+	config *LokiConfig
+	client *connector.LokiConnector
+	logger *logging.Logger
+	logChannel chan interface{}
+}
+
+//New constructor
+func New(logger *logging.Logger) application.Application {
+	return &Loki {
+		logger:     logger,
+		logChannel: make(chan interface{}, 100),
+	}
+}
+
+// ReceiveEvent ...
+func (l *Loki) ReceiveEvent(log data.Event) {
+	switch log.Type {
+	case data.LOG:
+		lokiLog, err := lib.CreateLokiLog(log)
+		if err != nil {
+			l.logger.Metadata(logging.Metadata{"plugin": "loki", "log": log, "error": err})
+			l.logger.Info("failed to parse the data in event bus - disregarding")
```

Comment:
Maybe this could be warning instead. If the parsing failure is also info in other apps, then ignore this :).

Reply:
I looked at other app plugins, and the only similar line to this I found was in [elastic search](https://github.com/infrawatch/sg-core/blob/90757690b2ab6a4495db8ad8e47d6c35eff99397/plugins/application/elasticsearch/main.go#L68). And it's an error there, so which one is better?

Reply:
If ES has error, let's keep consistency and have it as error here too.

Reply:
I changed both to Error